website/src/pages/tasks/random.tsx

```diff
@@ -0,0 +1,36 @@
+import { Container } from "@chakra-ui/react";
+import Head from "next/head";
+import { useEffect } from "react";
+import { LoadingScreen } from "src/components/Loading/LoadingScreen";
+import { Task } from "src/components/Tasks/Task";
+import { useGenericTaskAPI } from "src/hooks/tasks/useGenericTaskAPI";
+
+const RandomTask = () => {
+  const { tasks, isLoading, trigger, reset } = useGenericTaskAPI("random");
+
+  useEffect(() => {
+    if (tasks.length == 0) {
+      reset();
+    }
+  }, [tasks]);
+
+  if (isLoading) {
+    return <LoadingScreen text="Loading..." />;
+  }
+
+  if (tasks.length == 0) {
+    return <Container className="p-6 text-center text-gray-800">No tasks found...</Container>;
+  }
+
+  return (
+    <>
+      <Head>
+        <title>Random Task</title>
+        <meta name="description" content="Random Task." />
+      </Head>
+      <Task tasks={tasks} trigger={trigger} mutate={reset} />
```

Comment:
Someone changed this part a little bit.  It now takes `{tasks[0].task}`

Also, to fix some changes that just got merged in, there's a new property: `frontendId={tasks[0].id]}`

Reply:
For reference: https://github.com/LAION-AI/Open-Assistant/pull/593 fixes a few issues that with what's at HEAD.

Reply:
Got it, thanks. I'll take a look at it when I can (bit of a busy day at work today).

Reply:
Sounds good! Most of the fixes will be needed after I merge my PR anyway.

Reply:
Failed to rebase... I'll close this PR and open a new one I guess...